### DISCUSSION TITLE
How to use iframe in Markdown blog post?

### DISCUSSION THREAD

**Discussion by jonscrut** [datenstrom/community#615] [Ask a question] [answered]
Hi,
I am trying to embed an html page using iframe tag (long story, won't go into details).

I can't get it to work. I have a very simple html file as a test case which I have saved in the media folder in a sub-folder called "iframes".

`<iframe src="media/iframes/iframe1.html"> </iframe>`

However, when I use this code in blog post I get the Yellow "File not found" message.

I guess I am saving it in the wrong place or not referencing the file correctly.

Would appreciate some guidance :-)

**Comment by schulle4u** [comment, marked as answer]
You should add a slash before the iframe reference, this will use your website's document root as the correct starting point. 

`<iframe src="/media/iframes/iframe1.html"> </iframe>`

HTH

**Reply by jonscrut** [reply]
It was to do with my local development server. After your hint I resolved it by using relative path instead 

`<iframe src="../media/iframes/iframe1.html"> </iframe>`

Thanks!
